src/core/host_display.cpp

```diff
@@ -27,6 +28,9 @@ void HostDisplay::SetDisplayMaxFPS(float max_fps)
 
 bool HostDisplay::ShouldSkipDisplayingFrame()
 {
+  if (!m_display_duplicate_frames && !m_display_changed)
+    return true;
+
   if (m_display_frame_interval == 0.0f)
```

Comment:
Shouldnt the members on the side and left side of the equal be swapped. Also i dont think this is the way to compare floating point values  (diff less than machine precision)

Reply:
There's no mathematical operations happening on that value, its loaded straight from config. So an exact comparison is perfectly fine.